tests/integration/_cases/sourcemaps/sourcemaps-inject.trycmd

```diff
@@ -20,7 +23,6 @@ Ignored: The following sourcemap files already have debug ids
   - [..]-[..]-[..]-[..]-[..] - ./static/chunks/pages/asdf-05b39167abbe433b.js.map
 
 Ignored: The following source files don't have sourcemap references 
-  - ./server/chunks/flight-server-css-manifest.js
```

Comment:
why is this suddenly ignored?

Reply:
Before, we just processed all js files in the given directory. Now this goes through `SourceMapProcessor`, which has an internal map of source files that can be filtered by type. We only want to do the injection for minified files, and this file is (rightly or wrongly) not considered minified.

Reply:
Just looked at it, the file looks minified to me by virtue of only having a single very long line. I wonder if our heuristics should pick this up.

Reply:
Definitely should not be picked up :(

Reply:
For what it’s worth it’s better to assume minified than not minified for processing.

Reply:
If it *shouldn't* be picked up, then this PR is an improvement :)